Cython/Compiler/Code.py

```diff
@@ -1494,6 +1494,23 @@ def generate_string_constants(self):
             w = self.parts['pystring_table']
             w.putln("")
             w.putln("static __Pyx_StringTabEntry %s[] = {" % Naming.stringtab_cname)
+            if py_string.py3str_cstring:
```

Comment:
`py_string` is a left-over loop variable here that may or may not have been assigned.